pkg/nsm/apis/nsmconnect/nsmconnect.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package nsmconnect;
+
+import "github.com/ligato/networkservicemesh/netmesh/model/netmesh/netmesh.proto";
+
+message Label {
+    map<string ,string> selector = 1;
+}
+
+message Metadata {
+    string name = 1;
+    string namespace = 2;
+    Label labels = 3;
+}
+
+message InterfaceParameters {
+// Interface parameters go here.
+}
+
+message Interface {
+   InterfaceType type = 1;
+   Metadata metadata = 2;
+   InterfacePreference preference = 3;
+   InterfaceParameters parmeters = 4;
+}
+
+enum InterfaceType {
+    KERNEL_INTERFACE = 1;
+    VHOST_INTERFACE = 2;
+    MEM_INTERFACE = 3;
+    SRIOV_INTERFACE = 4;
+    HW_INTERFACE = 5;
+  }
+
+enum InterfacePreference {
+    DEFAULT = 0;
+    FIRST = 1;
+    SECOND = 2;
+    THIRD = 3;
+    FORTH = 4;
+    FIFTH = 5;
+}
+
+// ConnectionRequest is sent by a client or NSEP to build a connection.
+message ConnectionRequest {
+// Since connection request will trigger certain actions
```

Comment:
This is a pretty pedantic nit, but should we shift these comments to match the indentation below them?